lib/Dialect/CGGI/IR/CGGIOps.td

```diff
@@ -99,4 +98,16 @@ def CGGI_Lut3Op : CGGI_LutOp<"lut3", [AllTypesMatch<["a", "b", "c", "output"]>]>
   let results = (outs LWECiphertextLike:$output);
 }
 
+def CGGI_LutPreparedOp : CGGI_LutOp<"lut_prepared", [AllTypesMatch<["input", "output"]>]> {
```

Comment:
Should there be a verifier that checks that the LUT range is within the ciphertext's encoding's bitwidth? (Or we can leave as a TODO, since I think it might not even then be clear if we have some extra padding bits. But at least it is upper bound)

Reply:
Done